.github/workflows/ci.yml

```diff
@@ -278,14 +278,14 @@ jobs:
 
       - name: Publish installer package
         if: ${{ matrix.installer }}
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: SasView-Installer-${{ matrix.os }}-${{ matrix.python-version }}
           path: |
             installers/dist/setupSasView.exe
             installers/dist/SasView6.dmg
             installers/dist/sasview6.tar.gz
-          if-no-files-found: error
+          if-no-files-found: ignore
```

Comment:
Is this a good idea? Shouldn't the build be marked `failed` if the installer is not present?